packages/components/src/components/teaser/teaser.stories.ts

```diff
@@ -382,3 +382,361 @@ export const Parts = {
+ *     transition-duration: 150ms;
+ *     transition-duration: 300ms;
+ *     opacity: 0.5;
+ *   }
+ * }
+ * ```
+ */
+
+export const SamplesClickable = {
+  name: 'Samples: Clickable teaser',
+  parameters: {
+    controls: {
+      disable: true
+    },
+    backgrounds: {
+      default: 'white'
+    }
+  },
+  render: () => {
+    return html`
+      <style>
+        #teaserWithContentPlaceholder::part(media) {
+          flex-grow: 1;
+        }
+
+        sd-teaser.interactive:hover {
+          &::part(media) {
+            transition-property: opacity;
+            transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+            transition-duration: 150ms;
+            transition-duration: 300ms;
+            opacity: 0.5;
+          }
+        }
+      </style>
+      <div class="flex justify-between gap-8">
+        <a href="#" target="_blank" class="flex-1">
+          <sd-teaser variant="primary" breakpoint="9999" inset class="interactive">
+            <div slot="media" class="relative">
+              <img class="aspect-video object-cover" src="./placeholders/generic.jpg" alt="A generic placeholder jpg" />
+            </div>
+            <div slot="meta" class="meta-info">
+              <span class="meta-info-item">01.12.2013</span>
+              <span class="meta-info-item">| Author name</span>
+            </div>
+            <h3 slot="headline">Clickable teaser</h3>
+            <div class="flex flex-col gap-5">
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna aliqua.
+              </p>
+              <div class="flex-none">
+                <sd-button variant="primary" inverted>More</sd-button>
+              </div>
+            </div>
+          </sd-teaser>
+        </a>
+
+        <a href="#" target="_blank" class="flex-1">
+          <sd-teaser variant="white border-neutral-400" breakpoint="9999" inset class="interactive">
+            <div slot="media" class="relative">
+              <img class="aspect-video object-cover" src="./placeholders/generic.jpg" alt="A generic placeholder jpg" />
+            </div>
+            <div slot="meta" class="meta-info">
+              <span class="meta-info-item">01.12.2013</span>
+              <span class="meta-info-item">| Author name</span>
+            </div>
+            <h3 slot="headline">Clickable teaser</h3>
+            <div class="flex flex-col gap-5">
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna aliqua.
+              </p>
+              <div class="flex-none">
+                <sd-button variant="primary">More</sd-button>
+              </div>
+            </div>
+          </sd-teaser>
+        </a>
+      </div>
+    `;
+  }
+};
+
+/**
+ * ## Clickable teaser with extra links
```

Comment:
unfortunately this sample needs to be removed. We agreed not to provide a sample with nested links, as we dont want the users to use the teaser like this (UX, a11y - even though you found a workaround).

Reply:
at least I learned something while implementing ;) I think I didn't understand the ticket and the provided .gif a 100% then.